ReleaseProcess.md

```diff
@@ -0,0 +1,21 @@
+# Release process for OrderlyWeb
+
+1. Get the version of the OrderlyWeb repo that you want to release locally (typically the latest master branch).  
+1. Tag and build the release log by running 
+   `./scripts/release/make-release.py`
```

Comment:
Can we also document the `./scripts/release/tag-images.py` script here? And make the docker tagging protocol super explicit: `tag-images` will tag images with the newly created semantic version, but it will also (if `--local-only` is not passed) tag them with "release", which is referenced by the config in Montagu-Orderly-Web config .

But also, as I'm thinking through this, I'm thinking that my `--local-only` suggestion was a bad one! (sorry :woman_facepalming:). Because I guess we don't want to be too trigger happy with the "release" tag right? I'm now imagining the work flow to be more like

1. create new semantic version
2. run `./tag-images tag` which tags local docker images with the new semantic version
3. release to UAT with, e.g. `--extra=v0.1.0` (change the Montagu-Orderly-Web config to use our local registry for UAT/Science?)
4. possibly iterate on 1 -3, once release approved run `./tag-images publish` to push tags to docker hub and also to create the "release" tag
5. release to UAT in the normal way
6. sanity check
7. release to prod

Thoughts? We could also leave the script as is for now and see how we end up using it. But def document the whole "release" tag thing.

Reply:
OK, so it sounds like we want to go back to the way it was before, where we had an optional "tag" arg (which would tag the images and push to the private registry), and an optional "publish" (which would push to the public registry, and we're also going to put the 'release' tag on too at this point.